src/Microsoft.AspNetCore.SystemWebAdapters/Microsoft.AspNetCore.SystemWebAdapters.csproj

```diff
@@ -31,6 +31,8 @@
     <Compile Include="Adapters/SessionState/ISessionState.cs" />
     <Compile Include="IHtmlString.cs" />
     <Compile Include="HtmlString.cs" />
+
+    <PackageReference Include="System.Security.Principal.Windows" Version="4.3.0" />
```

Comment:
Why this version as opposed to a newer one (5.0.0 or 4.7.0)?

Reply:
not sure - the latest VS found was 4.3.0 - I"ll change it to 5.0.0